src/core/server/core_usage_data/core_usage_data_service.test.ts

```diff
@@ -382,11 +407,11 @@ describe('CoreUsageDataService', () => {
         });
 
         await expect(getConfigsUsageData()).resolves.toMatchInlineSnapshot(`
-          Object {
-            "pluginA.enabled": "[redacted]",
-            "pluginAB.enabled": "[redacted]",
-          }
-        `);
+                          Object {
+                            "pluginA.enabled": "[redacted]",
+                            "pluginAB.enabled": "[redacted]",
+                          }
+                      `);
```

Comment:
nit: fixing auto generated `toMatchInlineSnapshot` indentation in the file

Reply:
I removed most of the snapshot usages in the file